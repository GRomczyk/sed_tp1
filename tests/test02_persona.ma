[Top]
components : P1@Persona
out : out_port1 info_out1
in : in_port1 info

Link : in_port1 inPort1@P1
Link : info infoRequest@P1

Link : outPort1@P1 out_port1
Link : outInfo@P1 info_out1

[P1]
p : 1.0
influenciabilidad : 1.0
poder_convencimiento : 0.2
tiempo_sillon : 1.0
grado : 1
periodo_medio : 600000
