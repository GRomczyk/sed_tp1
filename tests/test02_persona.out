00:01:00:000:0 out_port1 [1, 0.2]
00:02:00:000:0 info_out1      1.00000
